webpack/assets/javascripts/react_app/components/common/forms/DateTime/DateTime.js

```diff
@@ -35,8 +36,8 @@ const DateTime = ({
       }
     >
       <DateTimePicker
-        hiddenValue={isRequired}
-        value={value}
+        hiddenValue={isBlank(value) && !isRequired}
+        value={!isBlank(value) ? value : new Date}
```

Comment:
I found out, we do not support null value, it is something would be great in the future.
No blocker though.